src/SourceBridgeClient.ts

```diff
@@ -20,15 +20,15 @@ export class SourceBridgeClient {
   }
 
   public sendEvent(message: EventEnvelope): void {
-    console.log('[SourceBridge] sending message to parent:', message)
+    console.log('[SourceBridge] sending message to parent', message)
     parent.postMessage(JSON.stringify(message), '*')
   }
 
   public async sendRequest<TResponse>(message: EventEnvelope): Promise<TResponse> {
     const promise = new Promise<TResponse>((resolve, _reject) => {
       this.openRequests.set(message.id, resolve as ResolveFn<unknown>)
     })
-    console.log('[SourceBridge] sending request to parent:', message)
+    console.log('[SourceBridge] sending request to parent ', message)
```

Comment:
[nit] Make the messages consistent, the above terminates without a trailing space